apps/cowswap-frontend/src/cowSdk.ts

```diff
@@ -8,6 +7,6 @@ const prodBaseUrls = process.env.REACT_APP_ORDER_BOOK_URLS
 
 export const metadataApiSDK = new MetadataApi()
 export const orderBookApi = new OrderBookApi({
-  env: isBarnBackendEnv ? 'staging' : 'prod',
+  env: 'prod',
```

Comment:
THIS HAS TO BE REVERTED!!!

Only for testing the prod behaviour, not visible against staging